invenio_oauthclient/models.py

```diff
@@ -129,6 +131,14 @@ class RemoteToken(db.Model, Timestamp):
     )
     """Access token to remote application."""
 
+    refresh_token = db.Column(
+        StringEncryptedType(type_in=db.Text, key=_secret_key), nullable=True
+    )
+    """Refresh token to remote application."""
+
+    expires = db.Column(db.DateTime, nullable=True)
```

Comment:
We use a slightly different form in [invenio-records](https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/models.py#L30), can you please have a look?